.travis.yml

```diff
@@ -4,11 +4,14 @@ cache: bundler
 rvm:
   - 2.1.8
   - 2.2.4
-  - 2.3.0
+  - 2.3.1
 gemfile:
   - Gemfile
   - gemfiles/rails-4.2.gemfile
+  - gemfiles/rails-master.gemfile
 matrix:
   exclude:
     - gemfile: Gemfile
       rvm: 2.1.8
+    - gemfile: gemfiles/rails-master.gemfile
+      rvm: 2.1.8
```

Comment:
Okay, so we're still testing our code with 2.1.8, just using another gemfile. I wasn't 100% sure how it all worked but I grok it now. Thanks :)